### ISSUE TITLE
Neat URL: your opinion in parallel with extension Redirector

### ISSUE THREAD

**Issue by OhSoGood** [Smile4ever/firefoxaddons#102]
Hi again Geoffrey

I was using ClearLink in the past, which is pitifully no more maintained. Looking for replacement,I first found Link Cleaner ( https://addons.mozilla.org/en-US/firefox/addon/link-cleaner/ ) but it fully hard-coded with no customization possible. Then I discovered your (also) excellent Neat Url and Redirector.

The Redirector extension ( https://addons.mozilla.org/fr/firefox/addon/redirector/ ) is intrinsically very close but complementary to Neat Url, they both manipulates the URLs to allow better privacy. Yet, having two extensions looks unoptimised, and mainly I'm afraid of potential conflict.

What's your opinion on it? Would there be any advantage of merging both ?

**Comment by Smile4ever** [comment]
Both were built with clean and short URLs in mind. In fact both also make a redirect to the short URL.

Stripping query parameters involves some logic that it not needed for simple redirecting.

My opinion: do one thing in an addon and do it well: Neat URL will clean query and hash parameters, and reduce URLs in length but not handle redirection. Handling redirection requires a different kind of user interface (like Redirector has).

I don't think there are advantages to merging both. See  #99
